src/lumigo_tracer/event/event_trigger.py

```diff
@@ -197,6 +197,10 @@ def _parse_streams(event: dict) -> Dict[str, str]:
         result.update(_parse_sqs_event(event))
     elif triggered_by == "kinesis":
         result["messageId"] = safe_get(event, ["Records", 0, "kinesis", "sequenceNumber"])
+        result["recordsNum"] = len(safe_get(event, ["Records"]))
+        event_id = safe_get(event, ["Records", 0, "eventID"])
+        if event_id:
+            result["shardId"] = event_id.split(":")[0]
```

Comment:
@uri-p do we index it in explore?

Reply:
Not by default, but I do want to add it in the future as an execution tag that gets index.
It should be the tradeoff between reducing the size of the event (those metadata fields are duplicates) and add those metadata fields as execution tags.

Reply:
So where exactly is it written now?

Reply:
They will be part of the trigger by information.
I want to add auto-tagging for shard id and we need to think about how to make it easy to use the bulk size.

You will be able at first to see the distribution of your invocation and shards, make it easy to understand that maybe your partition key is not optimized, currently you have to enable CW per shard metrics to find this information. 
This can be done with lumigo OOB

Reply:
So you'll add it to get transaction return value as well, right?

Reply:
It can be done, once we will decide who to present it in the UI we can easily expose it